### PR TITLE
Create LIT7047Qene.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I started to transcribe the text from Abbadie 145 as a first test. I hope that BNFabb145 is the correct shelfmark for it. I do not know what mode and what else should be encoded.

### CODE REVIEW

File: new/LIT7047Qene.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7047Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">በዘመን፡ ቀዳማዊ፡ ኢይሕሚ፡ ቋዕ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ba-zaman qadāmāwi ʾi-yǝḥǝmmi qʷāʿ</title>
+                <title xml:lang="en" corresp="#t1">In the old times the raven was not jealous</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT7047Qene.xml:10]: I would say: 

Ba-zaman qadāmāwi ʾi-yǝḥǝmmi qʷāʿ (Qǝne of wāzemā type)
In the old times the raven was not jealous ... 

(see below)

File: new/LIT7047Qene.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-08-27">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55rb.</note>
```
Comment (DenisNosnitsin1970) [inline new/LIT7047Qene.xml:56]: Locus to be used for fol. 55rb?

File: new/LIT7047Qene.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                <ab>
+                    <l n="1">በዘመን፡ ቀዳማዊ፡ ኢይሕሚ፡ ቋዕ፡</l>
```
Comment (DenisNosnitsin1970) [inline new/LIT7047Qene.xml:58]: If I understand correctly this is a qene of wāzemā type, with five verses, the first one being very long; 
a bit strange, but I saw already qenes with the 1st line disproportionally long. The division into verses is indicated with ።
Then we have as the first verse: በዘመን፡ ቀዳማዊ፡ ኢይሕሚ፡ ቋዕ፡ ኤልያስሃ፡ በተአዝዘቱ፡ አክራመ፡ ፫፡ እንበለ፡ መብልዕ፡ ወልብስ፡ ለዕንቋዕ፡ ቋዕ፡ ዓቅሙ፡ ንዑስ።

File: new/LIT7047Qene.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT7047Qene.xml:56]: I would add in the note: (division into verses is indicated).

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]
Thank you very much! 
Here are some remarks.

File: new/LIT7047Qene.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT7047Qene.xml:56]: > Locus to be used for fol. 55rb?

It seems to me, that `<locus/>` is not allowed in `<body/>`.

Screenshot 2024_locus in body

Comment (CarstenHoffmannMarburg) [reply new/LIT7047Qene.xml:56]: > I would add in the note: (division into verses is indicated).

I did not see indications of verses in the manuscript except for periods in 5 out of 9 verses/lines. Is it enough to make that statement?

Comment (DenisNosnitsin1970) [reply new/LIT7047Qene.xml]: Ok, let it be. There four ። signs used in this verse, they are indicated in your transcription, for five verses with ስ rhyme, and this is what we need for transcribing qene. Occassional rhymes (there is more verses ending with ስ, obviously), can be disregarded, if we have ። signs. The structure of the poem should be then arranges accordingly (5 verses); having this, it is easy fo define the type.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT7047Qene.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55rb. Division into verses is indicated with
+                    <foreign xml:lang="gez">።</foreign>. The end of the poem is indicated with <foreign xml:lang="gez">። ።</foreign>.</note>
+                <ab>
+                    <l n="1">በዘመን፡ ቀዳማዊ፡ ኢይሕሚ፡ ቋዕ፡ ኤልያስሃ፡ በተአዝዘቱ፡ አክራመ፡ ፫፡ እንበለ፡ መብልዕ፡ ወልብስ፡ ለዕንቋዕ፡ ቋዕ፡ ዘዓቅሙ፡ ንዑስ።</l>
+                    <l n="2">እስመ፡ ወሀብኮ፡ ዓይነከ፡ ማኅቶተ፡ ሥጋ፡ ግሡሥ።</l>
```
Comment (karljonaskarlsson) [inline new/LIT7047Qene.xml:61]: Maybe `ግ<del>ኁ</del>ሡሥ `?

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

**Review by karljonaskarlsson — APPROVED** [review]

**Pull request merged**